docs/topics/settings.rst

```diff
@@ -897,6 +897,16 @@ Default: ``'scrapy.core.scheduler.Scheduler'``
 
 The scheduler to use for crawling.
 
+.. setting:: SCHEDULER_DEBUG
+
+SCHEDULER_DEBUG
+---------------
+
+Default: ``False``
+
+Setting to ``True`` will log the first unserializable request encountered.
```

Comment:
`SCHEDULER_DEBUG` reads like "enable to help debugging the scheduler component."
Wording it as "it's only to log unserializable requests" feels a bit limiting.

What about something along the lines of
> enable this setting to log debug information about Scrapy requests scheduler.
> This currently logs if requests cannot be serialized to disk. If this happens, a line will appear in the logs (only once), and a `scheduler/unserializable` stats counter tracks how many times this happened.

Including an example log line would also be good.